packages/instant-apps-components/src/components/instant-apps-header/instant-apps-header.test.ts

```diff
@@ -0,0 +1,69 @@
+import { expect, test, describe } from 'vitest';
+import '../../../dist/components/instant-apps-header.js';
+
+describe('prop changes', () => {
+  const header = document.createElement('instant-apps-header');
+  header.setAttribute('background-color', '#151515');
+  test('custom title text', async () => {
+    const customTitle = 'Instant Apps Header';
+    header.titleText = customTitle;
+    document.body.appendChild(header);
+    await new Promise(resolve => requestIdleCallback(resolve));
+
+    expect(header.titleText).toBe(customTitle);
+  });
+
+  test('info button visibility', async () => {
+    header.setAttribute('info-button', 'true');
+    header.infoTitleText = 'info test';
+    document.body.appendChild(header);
+    await new Promise(resolve => requestIdleCallback(resolve));
+    header.toggleInfo();
+    expect(header.infoButton).toBe(true);
+    expect(header.infoIsOpen).toBe(true);
+  });
+
+  test('info button toggle', async () => {
+    header.toggleInfo();
+    expect(header.infoIsOpen).toBe(false);
+  });
+
+  test('info text title prop', async () => {
+    const infoElement = header.shadowRoot?.getElementById('infoButton');
+    expect(infoElement).toBeTruthy();
+    const infosTitle = infoElement?.getAttribute('title');
+    expect(infosTitle).toBe(header.infoTitleText);
+  });
+
+  test('logo scale', async () => {
```

Comment:
the logo scale tests aren't really testing anything other than the attribute value at the host element. Can you query into the dom that's rendered to check that the logo scale value is actually being used? the logo scale value is used as within a class name

https://github.com/Esri/instant-apps-components/blob/master/packages/instant-apps-components/src/components/instant-apps-header/instant-apps-header.tsx#L203

This applies to all other tests (landing page, social share, etc.) that are setting a property and just testing the value at the host element. Try querying for the elements to see that the correct JSX is being rendered.